### PR TITLE
Fix create material for printers that use 1.75mm materials

### PR DESCRIPTION
**Pull request by fieldOfView**
This PR is a followup to #1757. It fixes the problem that if the user has a 1.75mm printer active, the Create button would clone the 2.85mm generic PLA which would then be invisible for the current machine.

The fix is to clone the generic PLA material for the current approximate material diameter instead.

### CODE REVIEW

**Comment by nallath** [comment]
CURA-3828

**Comment by fieldOfView** [comment]
Don't worry, I'll get you to the CURA-4000 for the 2.6 release...

File: cura/Settings/ContainerManager.py [hunk 1]
```diff
@@ -712,14 +712,21 @@ def createMaterial(self) -> str:
         if not global_stack:
             return ""
 
-        containers = self._container_registry.findInstanceContainers(id="generic_pla")
+        approximate_diameter = round(global_stack.getProperty("material_diameter","value"))
+        containers = self._container_registry.findInstanceContainers(id="generic_pla*", approximate_diameter=approximate_diameter)
```
Comment (Ghostkeeper) [inline cura/Settings/ContainerManager.py]: A few code style problems here. I fixed them for you in f65043ecc8f7b506464b31ba5995fac58b4d0358.

**Comment by jackha** [comment]
Ahhh I missed the boat. In the process of reviewing and testing I heard my name a few times and now it's merged!

**Comment by Ghostkeeper** [comment]
:boat:

**Pull request merged**